script_facade/app.py

```diff
@@ -10,15 +12,39 @@ def create_app(testing=False, cli=False):
     app.config.from_object('script_facade.client.config')
     register_blueprints(app)
     configure_app(app)
+    configure_cache(app)
 
     return app
 
+
 def register_blueprints(app):
     """register all blueprints for application
     """
     app.register_blueprint(api.fhir.blueprint)
+    app.register_blueprint(api.misc.blueprint)
+
 
 def configure_app(app):
     """Load successive configs - overriding defaults"""
 
     app.config.from_object('script_facade.config')
+
+
+def configure_cache(app):
+    """Configure caching libraries"""
```

Comment:
Should this exit early if `REQUEST_CACHE_URL` isn't configured? Would prefer not to require redis for every deploy (eg testing), not a big deal though